server/plugin/command.go

```diff
@@ -345,6 +349,12 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 			}
 			subOrgMsg += "\n\n" + fmt.Sprintf("Notifications are disabled for %s", excludeMsg)
 		}
+		githubHooks, _, _ := githubClient.Repositories.ListHooks(ctx, owner, repo, &github.ListOptions{PerPage: page})
+		if len(githubHooks) == 0 {
+			subOrgMsg += "\n No webhook was found for this repository or organization. Would you like the webhook to be created?"
```

Comment:
I don't think we want the spaces after each `\n`. Can you go through the PR and remove the extra space at the beginning of each new line?

```suggestion
			subOrgMsg += "\nNo webhook was found for this repository or organization. Would you like the webhook to be created?"
```

Reply:
Done